modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,244 @@
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String |  The object type name |  | *x*
+| Config Ref | ConfigurationProvider |  The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Redelivery Policy | <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  | 
+| Body Content | Any |  |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Array of Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
+
+== Types
+
+=== Data Fetcher Attributes
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Alias | String |  |  | 
+| Arguments | Object |  |  | 
+| Selected Fields | Object |  |  | 
+| Source | Any |  |  | 
+|======================
+
+=== Repeatable In Memory Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size | Number | This is the amount of memory that is allocated to consume the stream and provide random access to it. If the stream contains more data than buffer capacity, then it is expanded according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
+| Buffer Size Increment | Number | This is by how much buffer size is expanded if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  | 
+| Max Buffer Size | Number | This is the maximum amount of memory used. If more is used, a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower or equal to zero means no limit. |  | 
+| Buffer Unit a| Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB 
+
+| These attributes are expressed in this unit. |  | 
+|======================
+
+=== Repeatable File Store Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| In Memory Size | Number | Defines the maximum memory that the stream should use to keep data in memory. If more is consumed, it starts to buffer the content on disk. |  | 
+| Buffer Unit a| Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB  
+
+| The unit in which maxInMemorySize is expressed |  | 
+|======================
+
+=== Redelivery Policy
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering a "Process failed" message. |  | 
+| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  | 
+| Message Identifier | <<RedeliveryPolicyMessageIdentifier>> | Defines which strategy is used to identify the messages. |  | 
```

Comment:
Should it link to "=== Redelivery Policy Message Identifier" title?
If so, we link to content with an xref: xref:filename.adoc#articletitle[title or phrase to explain title action]

Reply:
Yes

Reply:
OK! Thanks!